assets/src/application/ui/display/entityDetailsPanel/EntityDetail.tsx

```diff
@@ -7,10 +7,11 @@ const EntityDetail: React.FC<EntityDetailProps> = ({ render: Component, ...props
 		return <Component {...props} />;
 	}
 	const { id, label, value, className = '' } = props;
+
 	return (
-		<div className={`ee-entity-details-div ${id}-div ${className}`}>
-			<div className={`ee-entity-details-label-div ${id}-label`}>{label}</div>
-			<div className={`ee-entity-details-value-div ${id}-value`}>{value}</div>
+		<div className={`ee-entity-details ${id}-div ${className}`}>
```

Comment:
not sure what was the intention behind `${id}-div`, but left that there for the time being